CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+0.0.13
+------
+
+* Bug fix: TimeoutError hanlding improvement (#109).
```

Comment:
Let's replace this line with:

```
* Bug fix: return RequestTimeoutError and ServerConnectionError exceptions instead of AuthError, no longer raise KeyError when a file fails to download via the proxy (#109)
```

Reply:
No line break or just copy pasting the text you wrote?

Reply:
oh this was a comment just on the text itself